src/components/colorPicker/ColorPickerDialog.tsx

```diff
@@ -154,15 +182,16 @@ class ColorPickerDialog extends PureComponent {
     }
   };
 
-  updateColor(color) {
+  updateColor(color: string) {
     const hex = this.getHexString(color);
     const text = this.getColorValue(hex);
     this.setState({color, text, valid: true});
   }
 
   resetValues() {
-    const color = Colors.getHSL(this.props.initialColor);
-    const text = this.getColorValue(this.props.initialColor);
+    const {initialColor} = this.props;
+    const color = initialColor && Colors.getHSL(initialColor);
```

Comment:
move undefined check to the getHSL method

Reply:
As I mentioned before, I don't think changing this implementation should be in this PR scop as this is not a ColorPickerDialog's method but Color's, which other components use.

Reply:
See my previous comment, it's a small change in `Colors`. No reason not to include it.

Reply:
Ok

Reply:
Done